app/javascript/components/patient/close_contacts/CloseContactTable.js

```diff
@@ -331,13 +330,11 @@ class CloseContactTable extends React.Component {
           contact_attempts: ccData.contact_attempts || 0,
         })
         .then(() => {
-          this.setState({ isLoading: false });
           // In the future this should be change to call a callback and update the UI without the need
           // for a page reload
           location.reload();
         })
         .catch(error => {
-          this.setState({ isLoading: false });
           reportError(error);
```

Comment:
This should be put back. In this case we do want to stop the spinner, because there is nothing to load after this point.

Reply:
I can see the case for either here, but this also determines disabling the button and we don't want to reenable it so the user can spam the server with the same bad request

Reply:
Not following. What button? The modal is dismissed here

Reply:
my bad I was totally thinking about the modal. this does need to stay

Reply:
Sounds good, re-adding it now!